src/constants/constants.ts

```diff
@@ -2,5 +2,6 @@ export const QUERY_PAR_LANG = "lang"
 export const NODE_MODULES_LOCALES_PATH = 'node_modules/@companieshouse/ch-node-utils/locales';
 export const SHARED_NAVBAR_NAMESPACE = "ch-node-utils-navbar"
 export const SHARED_NUNJUCKS_TRANSLATION_NAMESPACES = new Set([
-  SHARED_NAVBAR_NAMESPACE
-]);
\ No newline at end of file
+    SHARED_NAVBAR_NAMESPACE
+]);
+export const ACSP_NUMBER_REGEX_PATTERN = "^[0-9A-Za-z-_]{0,32}$";
```

Comment:
You might export directly the regex object instead of the pattern string. This would not need a `new RegeXp(....)` later on. Also, with the `/i` you can collapse the case sensitivines. In the end something like this should work:
```javascript
export const ACSP_NUMBER_REGEX = /^[0-9A-Z-_]{0,32}$/i;
```

Reply:
Done